packages/verification-vue-component/src/components/Verification/VerificationDropBox.vue

```diff
@@ -1,53 +1,47 @@
 <template>
-    <div class="verification-dropbox" :class="{dragover:draggingOver}"
-         @dragover.prevent="draggingOver = true"
-         @dragleave="draggingOver = false"
-         @drop="handleDrop">
+    <div class="verification-dropbox" @drop="handleDrop">
         <div class="dropbox">
             <div class="content">
-                <img src="../../assets/img/dropbox_document.svg" alt="Certifaction"/>
                 <div class="labels">
                     <div class="label title">{{ titleLabel }}</div>
-                    <div class="label subtitle">{{ _$t('verification.dropbox.subtitle') }}</div>
+                    <div class="label subtitle">{{ _$t('verification.fileSelection.subtitle.dropbox') }}</div>
                 </div>
+                <img src="../../assets/img/dropbox_document.svg" alt="Certifaction"/>
             </div>
             <input class="input-file"
                    type="file"
                    multiple
                    name="uploadField"
                    accept="application/pdf"
-                   @change="filesDropped($event.target.name, $event.target.files)">
+                   @change="filesSelected($event.target.name, $event.target.files)">
```

Comment:
Is that still needed in the dropbox component?

Reply:
The dragged file needs to be dropped somewhere in. Without it the dropped file just opens in the browser and doesn't verify. Do you know an alternative?

Reply:
The drop should be catched by the event, the file input shouldn't be needed on this level.

Reply:
You are right. I had here a wrong thought, changed it acc.